README.md

```diff
@@ -22,15 +22,98 @@ At this point, the CLI is not published as a package yet and needs to be install
 cargo install --git https://github.com/TheBevyFlock/bevy_cli --locked bevy_cli
 ```
 
-## Logging
+The **linter** currently needs to be installed separately with a specific nightly toolchain:
 
-The default logging level for the CLI is set to `info`. To change the log level set the `BEVY_LOG` environment variable.
+```sh
+# Install the toolchain
+rustup toolchain install nightly-2025-04-03 \
+    --component rustc-dev \
+    --component llvm-tools-preview
+
+# Install bevy_lint
+cargo +nightly-2025-04-03 install \
+    --git https://github.com/TheBevyFlock/bevy_cli.git \
+    --tag lint-v0.3.0 \
+    --locked \
+    bevy_lint
+```
+
+## Quick Start
+
+With the following steps, you can create a new 2D app with Bevy and run it in your browser:
+
+1. Create a new Bevy app using the 2D template:
+
+    ```sh
+    bevy new -t=2d my_bevy_app
+    ```
+
+2. Navigate into the folder:
+
+   ```sh
+   cd my_bevy_app
+   ```
+
+3. Check the code quality with the linter:
+
+    ```sh
+    bevy lint
+    ```
+
+4. Run the app in the browser:
+
+    ```sh
+    bevy run web --open
+    ```
+
+## Scaffolding
+
+The `bevy new` command lets you easily scaffold a new Bevy project using a custom template or a [minimal template provided by Bevy](https://github.com/TheBevyFlock/bevy_new_minimal).
+Templates are just GitHub repositories and can be specified with the `-t` flag.
+
+### Usage
+
+If the template is omitted, the [default minimal template](https://github.com/TheBevyFlock/bevy_new_minimal) will be chosen.
 
 ```sh
-export BEVY_LOG=trace
+bevy new my-project
 ```
 
-## Bevy web apps
+To use a specific template, provide the full GitHub URL
+
+```sh
+bevy new my-project -t https://github.com/TheBevyFlock/bevy_new_2d
+```
+
+Additionally, any repo prefixed with `bevy_new_` from the [TheBevyFlock](https://github.com/TheBevyFlock) will be usable via its shortcut form i.e. `-t 2d` will use the template [bevy_new_2d](https://github.com/TheBevyFlock/bevy_new_2d).
+
+```sh
+bevy new my-project -t 2d
+```
+
+## Linter
+
+The CLI has 1st-party support for `bevy_lint`, the static analysis tool that checks over your code (similar to Clippy!). It must be installed first using the [installation guide], but then you can run the linter with the `lint` subcommand:
+
+```sh
+bevy lint
+```
+
+This command uses the same arguments as `cargo check`:
+
+```sh
+bevy lint --workspace --all-features
+```
+
+You can view a full list of supported options with:
+
+```sh
+bevy lint -- --help
+```
+
+[installation guide]: https://thebevyflock.github.io/bevy_cli/bevy_lint/index.html#installation
+
+## Web apps
 
 The CLI makes it easy to build and run web apps made with Bevy, using `bevy build web` and `bevy run web`.
 It takes care of compiling the app to Wasm, creating JavaScript bindings and serving it on a local web server to test it out.
```

Comment:
(Nit) I'm unable to comment on it, but below this line it says "Necessary tools will also be installed automatically." I worry this will make users think the CLI will automatically install programs without asking them first. Could you switch it up to something like "The CLI is able to install missing tools for you, and will prompt you if it one is absent."